### PR TITLE
[zoneminder] Add support for setting the server run state

### PR DESCRIPTION
**Pull request by mhilbush**
Adds a new channel (runState) on the bridge/server thing that is used to set the run state of the ZoneMinder server.

Reference https://community.openhab.org/t/change-zoneminder-run-states/51178

Signed-off-by: Mark Hilbush <mark@hilbush.com>

### CODE REVIEW

File: bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/thing/thing-types.xml [hunk 1]
```diff
@@ -22,7 +23,15 @@
 			<channel id="videoUrl" typeId="url">
 				<label>Video URL</label>
 			</channel>
+			<channel id="runState" typeId="runState">
+				<label>Run State</label>
```
Comment (jlaur) [inline bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/thing/thing-types.xml:27]: This label is redundant as it's the same as defined in the channel type.

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! Only one minor comment.

File: bundles/org.openhab.binding.weathercompany/src/main/resources/OH-INF/update/instructions.xml
Comment (jlaur) [inline bundles/org.openhab.binding.weathercompany/src/main/resources/OH-INF/update/instructions.xml:1]: It seems this file ended up in the wrong binding.

File: bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/thing/thing-types.xml (hunk 1 above)
Comment (mhilbush) [reply bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/thing/thing-types.xml]: Fixed

File: bundles/org.openhab.binding.weathercompany/src/main/resources/OH-INF/update/instructions.xml
Comment (mhilbush) [reply bundles/org.openhab.binding.weathercompany/src/main/resources/OH-INF/update/instructions.xml:1]: Ugh. The file looked right but I didn't notice the path. Fixed.

File: bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/i18n/zoneminder.properties [hunk 2]
```diff
@@ -111,3 +113,7 @@ channel-type.zoneminder.triggerAlarm.label = Trigger Alarm
 channel-type.zoneminder.triggerAlarm.description = Triggers an alarm
 channel-type.zoneminder.url.label = URL
 channel-type.zoneminder.url.description = URL of image or stream
+
+# thing types
+
+thing-type.zoneminder.server.channel.runState.label = Run State
```
Comment (jlaur) [inline bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/i18n/zoneminder.properties:119]: This can now be removed.

Comment (mhilbush) [reply bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/i18n/zoneminder.properties:119]: Hmm. That's strange. I reran `mvn i18n:generate-default-translations`. Shouldn't that have removed it?

Comment (jlaur) [reply bundles/org.openhab.binding.zoneminder/src/main/resources/OH-INF/i18n/zoneminder.properties]: It doesn't remove strings, it only moves to the bottom those that are not statically declared.

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**